src/App/Fossa/Config/Common.hs

```diff
@@ -184,7 +184,11 @@ validateApiKey maybeConfigFile EnvVars{envApiKey} CommonOpts{optAPIKey} = do
       optAPIKey
         <|> (maybeConfigFile >>= configApiKey)
         <|> envApiKey
-  pure $ ApiKey textkey
+  if Data.Text.null textkey
```

Comment:
This ignores the case of `textKey` where the only character is space. 

Let's strip the text for spaces on the left-hand side, and right-hand side as well. 

```haskell
if Data.Text.null . Data.Text.strip $ textkey 
then fatalText "An empty FOSSA API key was specified. Please provide a non-empty FOSSA API key."
else pure $ ApiKey textkey
```

Reply:
Also, you imported `null` on line 68, so instead of `Data.Text.null`, you can just use `null`.  That may give you a problem if some other module has imported `null` somewhere else (it's a common name, I wouldn't be surprised), but if it compiles with no issue then it's fine.

Reply:
It was yelling at me in VS Code so I put it, I don't think I would have had I been using vim, I typically opt for shortest possible naming